docs/preview/clients/python/relational_api.md

```diff
@@ -1,314 +1,6045 @@
 ---
 layout: docu
+redirect_from:
+- /docs/api/python/relational_api
+- /docs/api/python/relational_api/
+- /docs/clients/python/relational_api
```

Comment:
This introduces conflicting redirects with the stable version, which has the same redirects.
I propose to remove the redirects from here.

When we move the preview version to the stable versions, we'll apply the redirects again, so these links (which are now printed in books and used by other websites) point to the latest stable version of DuckDB.